src/platforms/python/common/performance/index.mdx

```diff
@@ -3,7 +3,26 @@ title: Performance Monitoring
 description: "Learn more about how to configure our Performance integrations to get the best experience out of it."
 ---
 
-Performance Monitoring helps you see everything from macro-level metrics to micro-level spans, and you’ll be able to cross-reference transactions with related issues, customize queries based on your personal needs, and substantially more and is avaliable for the Sentry Python SDK version ≥ 0.11.2.
+Sentry allows you to monitor the performance of your application, showing you how latency in one service may affect another service, and letting you pinpoint exactly which parts of a given operation may be responsible. To do this, it captures distributed traces consisting of transactions and spans, which measure individual services and individual operations within those services, respectively. You can learn more about this model in our [Distributed Tracing](/product/performance/distributed-tracing/) docs. Performance Monitoring is available for the Sentry Python SDK version ≥ 0.11.2.
+
+## Enabling Tracing
+
+To get started automatically sending traces you can set the `traces_sample_rate` in your SDK config. This should be a number between `0` and `1`. (For example, to send 20% of transactions, set traces_sample_rate to `0.2`.)
+
+<PlatformContent includePath="getting-started-config" />
```

Comment:
Two things here:

1) While this is true, now that there's the `traces_sampler`, perhaps you could include more of what's here in the above? https://github.com/getsentry/sentry-docs/blob/4d159bbbbf325dde724c6f4a3152c020b0374cee/src/platforms/javascript/common/performance/index.mdx#L38-L85

2) Relatedly, I don't think this is the include we want. Can you please use this instead? `<PlatformContent includePath="performance/enable-tracing" />`. You'll need to create the `enable-tracing` folder in `includes/performance` (it exists already, but only on my branch), so that way once my work is done, your include will already be in the right place?

Reply:
@lobsterkatie I don't see `enable-tracing` in any other PR. Can you link me to it? What content should go in here?

Reply:
No PR yet - just in a branch. Might be easier for me to just pull your branch and add what I'm talking about. Would you mind?

Reply:
(To answer your question, though, a Python version of the code above.)

Reply:
No PR yet - just in a branch. To answer your question, though, a Python version of the code above.

Basically this:

```
` ` `python
import sentry_sdk

def traces_sampler(sampling context):
    # ...

sentry_sdk.init(
  dsn="___PUBLIC_DSN___",

  # To set a uniform sample rate
  traces_sample_rate=0.2,

  # Alternatively, to control sampling dynamically
  traces_sampler=traces_sampler
)
` ` `
```

except remove the spaces from in between the backticks. I had to put them there to make them render and not get used by the GH MD engine.